packages/eslint-config-sdk/src/index.js

```diff
@@ -164,7 +164,7 @@ module.exports = {
       env: {
         jest: true,
       },
-      files: ['*.test.ts', '*.test.tsx', '*.test.js', '*.test.jsx', 'test/**/*.ts', 'test/**/*.js'],
+      files: ['test.ts', '*.test.ts', '*.test.tsx', '*.test.js', '*.test.jsx', 'test/**/*.ts', 'test/**/*.js'],
```

Comment:
Why do we need this change?

Reply:
Our tests under `suites` have the name `test.ts` and `@typescript-eslint/no-unsafe-member-access` gives false-positives (only observed that rule, but there might be problems with other rules defined here, when we add more tests.). Rather than overriding rules locally, I thought it would be better to add these tests to the scope of `eslint-config-sdk`. Would also do `suites/**/test.ts`, if that makes more sense.